values.yaml

```diff
@@ -1,4 +1,7 @@
 georchestra:
+  # Set additional nodeSelector items
```

Comment:
I think you can set it like this instead:
```
# Set additional nodeSelector items
nodeSelector: {}
#  "georchestra": "true"
```
Which is more common for empty arrays by default.

Reply:
fine for me. Thanks